2089_find_target_indices.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+
+class Solution:
+    def targetIndices(self, nums: List[int], target: int) -> List[int]:
+        
+        nums.sort()
+        matrix = []
```

Comment:
This variable name doesn't correspond to the usage of the variable

Reply:
I changed the name from matrix into index